.github/workflows/ci.yml

```diff
@@ -18,31 +18,29 @@ on:
   workflow_dispatch:
 
 jobs:
-  macOS:
-    runs-on: macOS-13
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v3
-    - name: SPM tests
-      run: swift test --enable-code-coverage
-    - name: Convert coverage files
-      run: |
-        xcrun llvm-cov export -format "lcov" \
-          .build/debug/hummingbirdPackageTests.xctest/Contents/MacOs/hummingbirdPackageTests \
-          -ignore-filename-regex="\/Tests\/" \
-          -ignore-filename-regex="\/Benchmarks\/" \
-          -instr-profile=.build/debug/codecov/default.profdata > info.lcov
-    - name: Upload to codecov.io
-      uses: codecov/codecov-action@v3
-      with:
-        file: info.lcov
+#  macOS:
```

Comment:
Why don't we have macOS CI?

Reply:
Requires 5.9 and there isnt a macOS14 runner.